src/pages/signin/index.js

```diff
@@ -0,0 +1,26 @@
+import { useTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+import SignInForm from "../../components/SignInForm";
+import SittingImg from "../../components/SittingImg";
```

Comment:
And here you can use the `@/components/SignInForm` which will be better than the dots.